cookbooks/tomography_based_plate_motions/plugins/reference_profile.h

```diff
@@ -0,0 +1,207 @@
+/*
+  Copyright (C) 2011 - 2023 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_cookbooks_tomography_based_plate_motions_reference_profile_h
+#define _aspect_cookbooks_tomography_based_plate_motions_reference_profile_h
+
+
+#include <aspect/adiabatic_conditions/interface.h>
+
+#include <deal.II/base/parsed_function.h>
+
+namespace aspect
+{
+  namespace AdiabaticConditions
+  {
+    using namespace dealii;
+
+    /**
+     * A model in which the adiabatic profile is
+     * calculated by solving the hydrostatic equations for
+     * pressure and temperature in depth.
+     * The gravity is assumed to be in depth direction
+     * and the composition is either given by the initial
+     * composition at reference points or computed
+     * as a reference depth-function.
+     * All material parameters are computed by the
+     * material model plugin. The surface conditions are
+     * either constant or changing over time as prescribed
+     * by an user-provided function.
```

Comment:
I think this is the documentation of the original ComputeProfile plugin? If so, please update with a description of what this plugin does.